src/helpers.js

```diff
@@ -246,7 +246,14 @@ function updateField(field, requiredFields, node, formValues) {
  * @param {Set} accRequired - set of required field names gathered by traversing the tree
  * @returns {Object}
  */
-function processNode(node, formValues, formFields, accRequired = new Set()) {
+export function processNode({
+  node,
```

Comment:
may seem odd to change the function signature here, but it makes later parts a lot easier. 🙏